media-api/test/lib/ImagePersistenceReasonsTest.scala

```diff
@@ -36,6 +36,7 @@ class ImagePersistenceReasonsTest extends AnyFunSpec with Matchers {
     imgPersistenceReasons.reasons(imgWithLeases) shouldBe List("leases")
     val imgWithPersistedRootCollections = img.copy(collections = List(Collection.build(persistedRootCollections.tail, ActionData("testAuthor", now()))))
     imgPersistenceReasons.reasons(imgWithPersistedRootCollections) shouldBe List("persisted-collection")
+    //TODO consider a test for when all images in collections are persisted
```

Comment:
sounds a good idea to me, worth doing before merge?

Reply:
done, thanks for the reminder